designs/administrator/future-openshift/bulk-actions/index.md

```diff
@@ -5,40 +5,92 @@ version: 4.x
 
 # Bulk actions
 
-Bulk actions will be added to a number of pages in the OpenShift console.
+Bulk actions will be added to a number of list views in the OpenShift console. This feature will allow users to perform an action on multiple resources at once. The list of actions include adding labels, adding annotations and deleting resources.
 
+## Bulk actions in toolbar
+The bulk selector will be the first component in the toolbar on pages where bulk actions are available.
+**ADD IMAGE GINA**
```

Comment:
Should we keep these "ADD IMAGE GINA" in the MD file? Should we wait to merge this until they are added?